microservices/questions-repo/src/app.ts

```diff
@@ -22,10 +23,13 @@ function readEnvironmentVariables() {
  */
 function initialiseDatabase() {
   // The URI should look like "mongodb+srv://admin:<password>@<clusterxx.yyyyyy>.mongodb.net/test?retryWrites=true&w=majority"
-  const mongoDbUri = process.env.MONGO_URI;
+  var mongoDbUri = process.env.MONGO_URI;
   if (!mongoDbUri) {
     console.log("MongoDB atlas URI not found in environment");
   } else {
+    if (fs.existsSync(mongoDbUri)) {
+      mongoDbUri = fs.readFileSync(mongoDbUri, "utf8").trim();
+    }
```

Comment:
I think the code can be reordered to be more like the auth-server for clarity

i.e., 

```js
const mongoDbUri = fs.existsSync(process.env.MONGO_URI)
  ? fs.readFileSync(process.env.MONGO_URI, "utf8").trim()
  : process.env.MONGODB_URI;

// if `process.env.MONGODB_URI === undefined`, you can `console.error()` the error here
```